Tests/InfomaniakCoreTests/Chunking/ITRangeProviderGuts.swift

```diff
@@ -0,0 +1,168 @@
+/*
+ Infomaniak kDrive - iOS App
+ Copyright (C) 2023 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+@testable import InfomaniakCore
+import XCTest
+
+/// Integration Tests of the RangeProviderGuts
+final class ITRangeProviderGuts: XCTestCase {
+    // MARK: - readFileByteSize
+
+    let file = "Matterhorn_as_seen_from_Zermatt,_Wallis,_Switzerland,_2012_August,Wikimedia_Commons"
```

Comment:
It looks like the file "Matterhorn_as_seen_from_Zermatt…" was not copied with the integration tests. They are probably failiing

Reply:
We already had in under /Resources for other tests

Reply:
Perfect 🤝